tests/optimize/test_edge_cli.py

```diff
@@ -94,6 +94,7 @@ def test_edge_init(mocker, edge_conf) -> None:
     assert edge_cli.config == edge_conf
     assert edge_cli.config['stake_amount'] == 'unlimited'
     assert callable(edge_cli.edge.calculate)
+    assert edge_conf['strategy'].bot_started is True
```

Comment:
I get why this is failing, but I'm not sure how I would test this out for edge

Reply:
`assert edge_cli.strategy.bot_started is True` ?